### ISSUE TITLE
upgrade to 1.1 does not use rails database_service

### ISSUE THREAD

**Issue by pauloancheta** [DataDog/dd-trace-rb#2115]
**Current behaviour**

Given my configuration
```ruby
Datadog.configure do |c|
    c.tracing.instrument :rails, service_name: "myapp-web", cache_service: "myapp-cache", database_service: "myapp-postgres"
end
```

My APM services are named as:
- myapp-web
- myapp-cache
- postgres

**Expected behaviour**

My APM services should be named as:
- myapp-web
- myapp-cache
- myapp-postgres

**Steps to reproduce**
Upgrading to 1.1 from 0.49.0

**How does `ddtrace` help you?**

**Environment**

* **ddtrace version: 0.49 & 1.1** 
* **Configuration block (`Datadog.configure ...`):**
* **Ruby version: 2.7.6**
* **Operating system: Mac Monterey 12.2.1**
* **Relevant library versions:**

**Comment by danielmklein** [comment]
@pauloancheta is this actually fixed? I think we are seeing the same thing you reported.

**Comment by danielmklein** [comment]
Ah, I see -- seems like https://docs.datadoghq.com/tracing/setup_overview/setup/ruby/ could use some love. 😄

**Comment by pauloancheta** [comment]
Yeah, I was going to update it this afternoon. But not sure if I'd have time 😅

**Comment by ivoanjo** [comment]
Thanks @pauloancheta and @danielmklein for pointing out our gap! The amazing @jennchenn has now update the docs to include this detail (#2177) and the update should show up soon on the official docs :)
